src/devices/device_models/common/model_utils.jl

```diff
@@ -31,3 +31,7 @@ function _remove_underscore(original::Symbol)
 
     return parts[1]
 end
+
+function remove_char(s::String, char::String)
+    return replace(s, Regex("[$char]") => "")
```

Comment:
What do you think about replacing ":" with "-" or "_"?

Reply:
Also, I think it would be better to use the normal, non-Regex version of the function.

Reply:
The purpose is to have a function that can do several character removals and replacements. See 05912f8

Reply:
A docstring would make that clear. :)  Also, maybe the function should be plural.